src/components/NavBar/NavBar.tsx

```diff
@@ -1,24 +1,59 @@
-import React from 'react';
+import React, { useState, useRef } from 'react';
 import { Link } from '@reach/router';
 import {
   AppBar,
   Toolbar,
   Typography,
-  withStyles,
   Grid,
+  Hidden,
+  Theme,
+  withStyles,
+  WithStyles,
+  createStyles,
+  Button,
+  Drawer,
 } from '@material-ui/core';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+import { faInfoCircle, faBars } from '@fortawesome/free-solid-svg-icons';
 import { faGithub } from '@fortawesome/free-brands-svg-icons';
 
 import MenuItem from './MenuItem/MenuItem';
 import { categories } from './utils';
 
-export interface INavBarProps {
-  classes: { [key: string]: string };
-}
+const rightSideLinks = [
+  {
+    title: 'About',
+    icon: faInfoCircle,
+    link: '/',
+  },
+  {
+    title: 'Github',
+    icon: faGithub,
+    link: '/',
+  },
+];
+
+function NavBar({ classes }: NavBarProps) {
+  const [openMobileMenu, setOpenMobileMenu] = useState(false);
```

Comment:
Do you need this boolean state or could you just save off the anchor element?

Reply:
Oups the anchor wasn't needed so I get rid of it 👍🏽